app/routes.py

```diff
@@ -13,12 +13,35 @@
 from IPython import embed
 import json
 
+# route helpers
+def get_foods(args):
+    food_items = []
+    for food in args:
+        food_response = {'id': food.id, 'name': food.name, 'calories': food.calories}
+        food_items.append(food_response)
+    return (food_items)
+
+def posts_list(args):
+    post_items = []
+    for post in args:
+        post_response = {
+            'id': post.id,
+            'title': post.title,
+            'description': post.description,
+            'image_url': post.image_url,
+            'post_type': post.post_type,
+            'user_id': post.user_id
+        }
+        post_items.append(post_response)
+    return (post_items)
```

Comment:
Looks great! Maybe we should create a controller folder to hold some of the logic of these requests.